tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -1619,6 +1619,23 @@ def map(self, map_func, num_parallel_calls=None, deterministic=None):
     >>> list(d.as_numpy_iterator())
     [b'HELLO', b'WORLD']
 
+    3) Use `tf.numpy_function`, which also allows you to write arbitrary
+    Python code. Note here that `tf.py_function` accepts `tf.Tensor` whereas
+    `tf.numpy_function` accepts numpy arrays and returns only numpy arrays.
+    For example:
+
+    >>> d = tf.data.Dataset.from_tensor_slices(['hello', 'world'])
+    >>> def upper_case_fn(t: np.ndarray):
+    ...   return t.decode('utf-8').upper()
+    >>> d = d.map(lambda x: tf.numpy_function(func=upper_case_fn,
+    ...           inp=[x], Tout=tf.string))
+    >>> list(d.as_numpy_iterator())
+    [b'HELLO', b'WORLD']
+
+    Note the fact that the use of `tf.numpy_function` and `tf.py_function`
```

Comment:
thank you, please remove "the fact" ... so this should start with "Note that the use of ..."

Reply:
done